src/transformers/vueadstabletree.transformer.js

```diff
@@ -0,0 +1,56 @@
+import _ from 'lodash'
+import Family from '@/model/Family.model'
+import Task from '@/model/Task.model'
+
+export default class VueAdsTableTreeTransformer {
+
+  constructor () {
+
+  }
+
+  transform (graphqlFamilyProxies = []) {
+    return _.map(graphqlFamilyProxies, (familyProxy) => {
+      return this._recursivelyTransformFamily(familyProxy)
+    })
+  }
+
+  _recursivelyTransformFamily (familyProxy) {
+    const childrenFamilies = familyProxy.childFamilies.length > 0 ? _.map(familyProxy.childFamilies, this._recursivelyTransformFamily) : []
+    const childrenTasks = familyProxy.childTasks !== undefined && familyProxy.childTasks.length > 0 ? _.map(familyProxy.childTasks, this._transformTask) : []
```

Comment:
Fine to call map with zero-length array?

Reply:
Yup, it just returns an empty array (should be covered in the unit tests)